docs/types/power_types/modify_insomnia_ticks.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Modify Insomnia Ticks (Power Type)
+date: 2022-06-07
+---
+
+# Modify Insomnia Ticks
+
+[Power Type](../power_types.md)
+
+Modifies the amount of time it takes for phantoms to spawn right before spawning them, as long as a condition is met. 
```

Comment:
```suggestion
Modifies the amount of time (in ticks) it takes for phantoms to spawn relatively above the player.
```

Reply:
> Okay, so what that power type does, is it modifies the current amount of "time since rest" where it's used for phantom spawn calculation. That means, the power type does not modify the "time since last rest" statistic, it instead only modifies the effective "time since last rest".
What you did is multiplying it by an omega large number, which means the effective time since last rest is very high and therefore phantoms would spawn basically always. :D
If you don't want phantoms to spawn, you need to modify the value to be less than 72000 (as phantoms start having the chance to spawn when the time since last rest is larger than 72000 ticks):
```
"modifier": {
  "operation": "set_total",
  "value": 0
}
```
> Higher values > 72000 increase the chance for phantoms to spawn. 

>  Phantom spawns are evaluated every 60-120 seconds (random interval), and it is only applied when those are checked.
I'm not sure that this change is accurate.